simpletransformers/ner/ner_model.py

```diff
@@ -704,10 +718,16 @@ def train(
                         outputs = model(**inputs)
                         # model outputs are always tuple in pytorch-transformers (see doc)
                         loss = outputs[0]
+                        if self.loss_fct:
```

Comment:
maybe consider filtering some of this label, because not all tokens are taken into account during loss calculation (I mean padding etc)
Maybe something like original implementation?
```active_loss = attention_mask.view(-1) == 1
                active_logits = logits.view(-1, self.num_labels)
                active_labels = torch.where(
                    active_loss, labels.view(-1), torch.tensor(loss_fct.ignore_index).type_as(labels)
                )
                loss = loss_fct(active_logits, active_labels)```
https://github.com/huggingface/transformers/blob/master/src/transformers/models/roberta/modeling_roberta.py#L1402

Reply:
answer in [review](#pullrequestreview-702707312)